consensus/hotstuff/voteaggregator/vote_collectors.go

```diff
@@ -129,5 +136,13 @@ func (v *VoteCollectors) PruneUpToView(lowestRetainedView uint64) {
 			delete(v.collectors, w)
 		}
 	}
+	from := v.lowestRetainedView
 	v.lowestRetainedView = lowestRetainedView
+	sizeAfter := len(v.collectors)
+	v.lock.Unlock()
+
+	v.log.Debug().
+		Uint64("from", from).
+		Uint64("to", lowestRetainedView).
+		Msgf("pruned vote collectors, before: %d, after: %d", sizeBefore, sizeAfter)
```

Comment:
should we change the values for `sizeBefore` and `sizeAfter` to explicit key-value pairs, e.g. 

```suggestion
	v.log.Debug().
		Uint64("prior_lowest_retained_view", from).
		Uint64("lowest_retained_view", lowestRetainedView).
		Int("prior_size", sizeBefore).
		Int("size", sizeAfter).
		Msgf("pruned vote collectors")
```